dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -93,11 +93,18 @@ private static Options configureOptions() {
         "report-only-reachable",
         false,
         "To report only linkage errors reachable from entry point");
+    options.addOption(
+        "m",
+        "maven-repositories",
+        true,
+        "Maven repositories (separated by ',') to search dependencies");
```

Comment:
Added description for order.

Format is a URL.

Added description on Maven central is in the list by default.

So far I don't see use case of excluding Maven central but it would be easy to implement when it's needed.

Reply:
I came up with a use case where Maven Central is inaccessible. So added `no-maven-central` option.

Reply:
Would it be simpler to just have the default for -m be Maven Central, and if any repository is specified then Maven Central must be as well?